CefSharp/DefaultRequestHandler.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright © 2010-2017 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System.Security.Cryptography.X509Certificates;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Default implementation of <see cref="IRequestHandler"/>.
+    /// This class provides default implementations of the methods from <see cref="IRequestHandler"/>,
+    /// therefore providing a convenience base class for any custom request handler.
+    /// </summary>
+    public class DefaultRequestHandler : IRequestHandler
+    {
+        public virtual bool GetAuthCredentials(IWebBrowser browserControl, IBrowser browser, IFrame frame, bool isProxy, string host,
```

Comment:
Ideally the `xmldoc` would be copied from the parent `IRequestHandler`. `GhostDoc` is handy when it comes to this sort of thing.